docx/oxml/sdts.py

```diff
@@ -54,14 +76,16 @@ class CT_SdtContentBase(BaseOxmlElement):
     """
     p = ZeroOrMore('w:p')
     r = ZeroOrMore('w:r')
+    sdt = ZeroOrMore('w:sdt')
+    tbl = ZeroOrMore('w:tbl')
 
     def iter_runs(self):
         def walk(el):
             for child in el:
                 if child.tag == qn('w:r'):
                     yield child
                 elif child.tag in (qn('w:smartTag'),):
-                    yield from get_runs(child)                    
+                    yield from get_runs(child)
```

Comment:
@aleksandarbos Tests work fine but VS Code prompts me that get_runs is not defined. Do you have this warning?